users/admin.py

```diff
@@ -1,3 +1,8 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from users.models import User, Notification
+
 
 # Register your models here.
+admin.site.register(User, UserAdmin)
```

Comment:
Remove UserAdmin from the arguments here and the corresponding import. Its irrelevant for now. If we need to have  the other fields for user in the admin panel later on, we will add this but will need some other add-ons as well.

Reply:
I've removed that just now, please have a look.